src/routes/demographics/Content.svelte

```diff
@@ -65,12 +65,12 @@
       type: "single",
       columns: true,
       values: [
-        { key: "19-24", label: "19-24 years old" },
-        { key: "25-34", label: "25-34 years old" },
-        { key: "35-44", label: "35-44 years old" },
-        { key: "45-54", label: "45-54 years old" },
-        { key: "55-64", label: "55-64 years old" },
-        { key: "> 65", label: "65 years and older" },
+        { key: "19_24", label: "19-24 years old" },
```

Comment:
@hamilton this PR changes the keys here, to make it easier to work with them data collection-wise.